File: ejercicios parte 1/1)programas simples/7hora futura/horafutura.cpp
#include <iostream>
using namespace std;
int main(){
    int h_act, aumento, h_fut;

    cout<<"Hora actual: ";
    cin>>h_act;
    cout<<"Cantidad de horas: ";
    cin>>aumento;
    h_fut = (h_act+aumento) % 12;
    
    cout<<"La hora futura será: "<<h_fut<<endl;
    return 0;
}

File: ejercicios parte 1/1)programas simples/7hora futura/horafutura.java
import java.util.Scanner;

public class horafutura {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int hact, aumento, hfut;
            
            System.out.print("Ingrese la hora acutal: ");
            hact = scan.nextInt();
            System.out.print("Ingrese el aumento: ");
            aumento= scan.nextInt();
            hfut = (hact+aumento) %12;

            System.out.println("La futura será: "+hfut);
        }
        
    }    
}
